### ISSUE TITLE
Porting Wildfly quickstart to fabric8-quickstarts organization

### ISSUE THREAD

**Issue by oscerd** [fabric8io/ipaas-quickstarts#1405]
I noticed Widlfly-quickstart doesn't exist anymore in this repo but it wasn't moved to the new organization.

Maybe it's just a forgetfulness.

**Comment by oscerd** [comment]
News?

**Comment by jstrachan** [comment]
yeah, we have moved quickstarts into this github organisation so that they are stand alone projects; easier for folks to fork and use directly and easier to grok.
https://github.com/fabric8-quickstarts

e.g. the wildfly swarm quickstart is here now
https://github.com/fabric8-quickstarts/swarm-camel

**Comment by oscerd** [comment]
Yeah, I meant that I don't see that particular quickstart anymore.

The original commit was this one:

https://github.com/fabric8io/ipaas-quickstarts/commit/e54bc8d9d344af4f076a68c3da499591e49cf86b

**Comment by jstrachan** [comment]
looks like someone removed all quickstarts before we had a chance to migrate them! I'd been going through them iteratively moving them across; the WAR ones needed a bit of work so i'd left them here

**Comment by oscerd** [comment]
@jstrachan no problem! It's not so much important! We already have a lot of quickstarts!
